src/NJsonSchema/JsonSchema.cs

```diff
@@ -441,10 +441,14 @@ public JsonObjectType Type
         [JsonProperty("minProperties", DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
         public int MinProperties { get; set; }
 
-        /// <summary>Gets or sets a value indicating whether the schema is deprecated (Swagger and Open API only).</summary>
+        /// <summary>Gets or sets a value indicating whether the schema is deprecated (native in Open API 'deprecated', custom in Swagger 'x-deprecated').</summary>
```

Comment:
Change to "custom in Swagger/JSON Schema"

Reply:
Done.